src/source_process.c

```diff
@@ -166,6 +206,9 @@ static uint64_t f_snd_trm(uint64_t *c, const prim_ctx_t *ctx) {
   const uint64_t p = ctx->p, m = ctx->m;
 
   // active groups
+  // This is the indexs of the non zero elements of c
```

Comment:
@rafl do you mind tidying up the comments in this function. I never quite got my head around it